icon-two-common-models/src/main/resources/xsdSchemas/auth-models-1.xsd

```diff
@@ -30,8 +30,8 @@
             <xsd:element name="latestPhoto" nillable="true" type="xsd:string"/>

```

Comment:
latestPhoto, firstName, lastName and csNum are not nillable

Reply:
you might want to be mindful about others fields when copy-pasting, some aren't nillable

Reply:
I checked. yes, Auth has some non-nillable properties in it.